pkg/blobstore/configuration/cas_blob_replicator_creator.go

```diff
@@ -27,7 +27,7 @@ func NewCASBlobReplicatorCreator(grpcClientFactory grpc.ClientFactory) BlobRepli
 func (brc *casBlobReplicatorCreator) NewCustomBlobReplicator(configuration *pb.BlobReplicatorConfiguration, source blobstore.BlobAccess, sink BlobAccessInfo) (replication.BlobReplicator, error) {
 	switch mode := configuration.Mode.(type) {
 	case *pb.BlobReplicatorConfiguration_Deduplicating:
-		base, err := NewBlobReplicatorFromConfiguration(mode.Deduplicating, source, sink, brc)
+		base, err := NewBlobReplicatorFromConfiguration(mode.Deduplicating, source, sink, brc, "cas")
```

Comment:
Probably better if we move `GetStorageTypeName()` to implementations of `BlobReplicatorCreator`? Because then we can call that function from here as well.

Reply:
Good call, moved it there.